src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.cs

```diff
@@ -506,7 +506,9 @@ private async Task EnqueueProjectConfigurationChangeWorkItemAsync(ProjectChanges
                     !object.Equals(oldProject.AssemblyName, newProject.AssemblyName) ||
                     !object.Equals(oldProject.Name, newProject.Name) ||
                     !object.Equals(oldProject.AnalyzerOptions, newProject.AnalyzerOptions) ||
-                    !object.Equals(oldProject.DefaultNamespace, newProject.DefaultNamespace))
+                    !object.Equals(oldProject.DefaultNamespace, newProject.DefaultNamespace) ||
+                    !object.Equals(oldProject.OutputFilePath, newProject.OutputFilePath) ||
+                    !object.Equals(oldProject.OutputRefFilePath, newProject.OutputRefFilePath))
```

Comment:
We may need to change this again if it turns out that we need to change OutputFilePath back to the obj path and add an additional property for the 'bin' path... cc @jasonmalinowski

Reply:
Can we write this differently somehow so we *don't* have to keep updating this list? :-)

Reply:
sure.